go/noise/laplace_noise.go

```diff
@@ -164,14 +165,30 @@ func (laplace) DeltaForThreshold(l0Sensitivity int64, lInfSensitivity, epsilon,
 	return 1 - math.Pow(1-partitionDelta, float64(l0Sensitivity))
 }
 
-func (laplace) computeConfidenceIntervalInt64(noisedValue, l0Sensitivity, lInfSensitivity int64, epsilon, delta, confidenceLevel float64) (ConfidenceInterval, error) {
-	// TODO: Implement confidence interval computation.
-	return ConfidenceInterval{}, nil
+// computeConfidenceIntervalInt64 computes a confidence interval that contains the raw integer value x from which int64 noisedX
+// is computed with a probability greater or equal to 1 - alpha based on the specified laplace noise parameters.
+func (laplace) computeConfidenceIntervalInt64(noisedX, l0Sensitivity, lInfSensitivity int64, epsilon, delta, alpha float64) (ConfidenceInterval, error) {
+	if err := checkArgsConfidenceIntervalLaplace("computeConfidenceIntervalInt64 (Laplace)", l0Sensitivity, float64(lInfSensitivity), epsilon,
+		delta, alpha); err != nil {
+		err = fmt.Errorf("computeConfidenceIntervalInt64(noisedX %d, l0sensitivity %d, lInfSensitivity %d, epsilon %f,"+
+			"delta %e, alpha %f) checks failed with %v", noisedX, l0Sensitivity, lInfSensitivity, epsilon, delta, alpha, err)
```

Comment:
what is %e?

Reply:
Outputs float64 delta in scientific notation, It is used in addNoise(..) functions in Laplace and Gaussian:
https://github.com/google/differential-privacy/blob/1b1dc6639173c0a13613189ec21851604a4c7335/go/noise/laplace_noise.go#L63

I assumed because it is usually extremely small so its not representable using normal "%f"